docs/about_GettingStarted.md

```diff
@@ -0,0 +1,167 @@
+﻿---
+layout: default
+title: Getting Started
+type: about
+---
+
+# Getting Started
+
+## about_GettingStarted
+
+# SHORT DESCRIPTION
+
+Here is a super simple Polaris application to get you started
+
+```ps
+Import-Module Polaris
+New-PolarisGetRoute -Path "/" -Scriptblock {
+    $Response.Send('Hello World!');
+}
+
+Start-Polaris
+```
+
+The application starts a server listening on port 8080 (the default) for connections. It will respond to with "Hello World!" to requests to the root url (/) or **route** it will respond to every other request with a 404 not found.
+
+I can get a response from the server by either opening a browser to http://localhost:8080/ or running the following PowerShell command:
+
+**Command**
+
+```ps
+PS> Invoke-RestMethod -Method GET -Uri http://localhost:8080/
+```
+
+**Output**
+
+```
+Hello World!
+
+PS>
+```
+
+# LONG DESCRIPTION
+
+Here's something a bit more realistic for a super basic web application. Let's create two simple pages. One that displays running processes and one that displays running services.
+
+For the first one our route to the page will be "/Services" and we'll just want to display in a simple HTML table the Name, Display Name, and Status of all the services installed on my current machine.
+
+We'll create the Polaris route like this
+
+```ps
+New-PolarisGetRoute -Path "/Services" -Scriptblock {
+   $RunningServices = Get-Service | select Name,DisplayName,Status | ConvertTo-Html -Title "Services" | Out-String
+   $Response.SetContentType("text/html")
+   $Response.Send($RunningServices)
+}
+```
+
+We're leveraging PowerShell's built in ConvertTo-HTML command to generate a little simple HTML for us from the Get-Process command. We are then setting the content type on the resposne to **text/html** which tells the browser to render the text response as HTML instead of treating it like a regular text file. We're then sending the HTML generated by the Powershell command on.
```

Comment:
```suggestion
We're leveraging PowerShell's built in ConvertTo-HTML command to generate a little simple HTML for us from the Get-Process command. We are then setting the content type on the response to **text/html** which tells the browser to render the text response as HTML instead of treating it like a regular text file. We're then sending the HTML generated by the Powershell command on.
```